### PR TITLE
Add blog example for GraphStorm 0.2

### PR DESCRIPTION
**Pull request by zheng-da**
*Description of changes:*
This contains all code required for GraphStorm 0.2 blog example.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/mag/README.md [hunk 1]
```diff
@@ -44,3 +44,142 @@ python3 -m graphstorm.run.gs_link_prediction \
...
+			--model-encoder-type hgt
+```
+
+The accuracy of RGCN with the BERT model fine-tuned with link prediction is 57.86%,
+while the accuracy of HGT is 57.86%.
+
+We can also train a GNN model with the BERT model fine-tuned for predicting venues.
+
+```
+python3 -m graphstorm.run.gs_node_classification \
```
Comment (classicsong) [inline examples/mag/README.md:160]: What is the difference between this one and L132?

File: examples/mag/mag_bert_ft.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  basic:
+    backend: nccl
+    verbose: false
+    save_perf_results_path: null
+  lm:
+    lm_train_nodes: 128
+    lm_infer_batch_size: 32
+    freeze_lm_encoder_epochs: 0
+  gnn:
+    model_encoder_type: lm
```
Comment (classicsong) [inline examples/mag/mag_bert_ft.yaml:27]: We need to move model_encoder_type to `basic:`

File: examples/mag/mag_bert_nc.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    verbose: false
+    save_perf_results_path: null
+    use_mini_batch_infer: true
+  input:
+    restore_model_path: null
+  output:
+    save_model_path: null
+    save_embed_path: null
+  lm:
+    model_encoder_type: lm
```
Comment (classicsong) [inline examples/mag/mag_bert_nc.yaml:23]: ditto

File: examples/mag/mag_min_v0.2.json [hunk 4]
```diff
@@ -0,0 +1,97 @@
+{
```
Comment (classicsong) [inline examples/mag/mag_min_v0.2.json:1]: Is this file mentioned in README?

File: examples/mag/README.md (hunk 1 above)
Comment (zheng-da) [reply examples/mag/README.md:160]: this one trains GNN model with BERT embeddings fine-tuned with node classification.
L132 uses the BERT embeddings fine-tuned with link prediction.

**Review by classicsong — APPROVED** [review]
LGTM

**Pull request merged**